WakaTime.py

```diff
@@ -36,6 +38,11 @@
 _hb_processor = None
 _last_hb = None
 _filename = ''
+_default_always_overwrite_projectname=0
+_default_chars='1234567890._'
+_default_prefix=''
+_default_postfix=''
+_default_use_project_folder=0
```

Comment:
What is the purpose of these, if you have corresponding default class attributes in `WakaTimePreferences`?
It seems simpler to just move these values to the class attributes and remove the global constants.

Reply:
Since I "played" around with various default settings, it was easier to have it defined at the top of file.
Moved them into the class, but kept them separate. They will be evaluated for the help-texts too (`'<empty>'` vs `''`).

If this seems too complex just for this cosmetic effect, the values can easily go directly into the default values of the properties.